plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/widget/CameraServerWidget.java

```diff
@@ -0,0 +1,36 @@
+package edu.wpi.first.shuffleboard.plugin.cameraserver.widget;
+
+import edu.wpi.first.shuffleboard.api.widget.Description;
+import edu.wpi.first.shuffleboard.api.widget.ParametrizedController;
+import edu.wpi.first.shuffleboard.api.widget.SimpleAnnotatedWidget;
+import edu.wpi.first.shuffleboard.plugin.cameraserver.data.CameraServerData;
+
+import org.fxmisc.easybind.EasyBind;
+
+import javafx.fxml.FXML;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Pane;
+
+@Description(name = "Camera Stream", dataTypes = CameraServerData.class)
+@ParametrizedController("CameraServerWidget.fxml")
+public class CameraServerWidget extends SimpleAnnotatedWidget<CameraServerData> {
```

Comment:
Like CameraServerData, this might want to be renamed and put into the base plugin (StreamWidget?) so that non-cameraserver streams can also take advantage of it.

(unless the control stuff obviates this?)